Src/Console.cpp

```diff
@@ -0,0 +1,24 @@
+/// \file Console.cpp
+///
+/// \date Feb 6, 2016
+///
+/// \author Douglas Miller
+///
+
+#include "Console.h"
+
+Console::Console(int argc, char **argv)
```

Comment:
I was trying to make things more flexible. My thought was to be able to 
customize certain types of console, for example if we make a 
SocketConsole we could get IP address and port (to listen on) off the 
commandline.

On 02/07/2016 12:14 AM, Mark Garlanger wrote:
>
> In Src/Console.cpp 
> <https://github.com/mgarlanger/VirtualH89/pull/4#discussion_r52113049>:
>
> > @@ -0,0 +1,24 @@
> > +/// \file Console.cpp
> > +///
> > +/// \date Feb 6, 2016
> > +///
> > +/// \author Douglas Miller
> > +///
> > +
> > +#include "Console.h"
> > +
> > +Console::Console(int argc, char **argv)
>
> What's the plan for argc & argv?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mgarlanger/VirtualH89/pull/4/files#r52113049>.
>